packages/aws-lambda/src/ssm.js

```diff
@@ -103,34 +95,34 @@ module.exports.waitAndGetInstanaKey = callback => {
   if (fetchedValue) {
     return callback(null, fetchedValue);
   }
-  // CASE: Customer has set INSTANA_SSM_PARAM_NAME, but we were not able to fetch the value from AWS
+  // CASE: INSTANA_SSM_PARAM_NAME was set, but AWS response could not be fetched
   if (errorFromAWS) {
     return callback(errorFromAWS);
   }
 
   const endInMs = Date.now();
-  const awsTimeoutInMs = process.env.INSTANA_AWS_SSM_TIMEOUT_IN_MS ? Number(process.env.INSTANA_AWS_SSM_TIMEOUT) : 1000;
+  const awsTimeoutInMs = process.env.INSTANA_AWS_SSM_TIMEOUT_IN_MS
+    ? Number(process.env.INSTANA_AWS_SSM_TIMEOUT_IN_MS)
+    : 1000;
 
-  // CASE: the time between ssm lib initialisation and waitAndGetInstanaKey call
-  //       (which is the end of the customers lambda handler) is already too big to wait for the AWS response
+  // CASE: The time between SSM initialization and waitAndGetInstanaKey is too long to wait for the AWS response
   if (endInMs - initTimeoutInMs > awsTimeoutInMs) {
     return callback(`Stopped waiting for AWS SSM response after ${awsTimeoutInMs}ms.`);
   }
 
   /**
-   * Inside AWS the call to `getParameter` mostly takes 30-50ms
-   * Because we initialise the fetch already before the customer's handler is called,
-   * the chance is very high that the interval is not even used.
-   *
-   * In our tests it takes usually ~>150ms (remote call)
+   * The `GetParameterCommand` call in AWS typically takes about 80 to 500ms.
+   * This fetching process starts before the customer's handler is invoked,
+   * and a delay was noticed during a cold start, so this interval may be significant in that case.
+   * However, it's unlikely that this interval will be needed in other scenarios.
    */
-  let stopIntervalAfterMs = 250;
+  let stopIntervalAfterMs = 500;
```

Comment:
Increased the stopIntervalAfterMs, sometimes it seems the response took 80 to 500ms in tests. We can revisit this later.